crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -111,10 +123,19 @@ impl SizedBundle {
     }
 }
 
+pub(super) struct SizedSequenceAction(SequenceAction);
+
 #[derive(Debug, thiserror::Error)]
 pub(super) enum BundleFactoryError {
     #[error("sequence action is larger than the max bundle size. seq_action size: {size}")]
     SequenceActionTooLarge { size: usize, max_size: usize },
+    #[error(
+        "finished bundle queue is full and the sequence action does not fit in the current bundle"
+    )]
+    FinishedQueueFull {
+        curr_bundle_size: usize,
+        seq_action: SequenceAction,
```

Comment:
i think we should return the current size of the finished queue too here.

Reply:
If its full then its size is the capacity passed by config - this should be logged by the service on startup.